R/eonet.R

```diff
@@ -26,53 +41,57 @@ earth_event <- function(status = "all",
                         days = 20,
                         LimitType = "limit",
                         TrySimplify = TRUE)
-  {
+{
 
-  h <- "http://eonet.sci.gsfc.nasa.gov/api/v2.1/events?"
+  events_url <- "http://eonet.sci.gsfc.nasa.gov/api/v2.1/events?"
```

Comment:
I think we will need to add this variable as `events_url <- NULL` in corresponding file, to pass the checks